openmetadata-ui/src/main/resources/ui/src/pages/database-details/index.tsx

```diff
@@ -368,16 +397,22 @@ const DatabaseDetails: FunctionComponent = () => {
     return new Promise<void>((_, reject) => {
       saveUpdatedDatabaseData(updatedData as Database)
         .then((res: AxiosResponse) => {
-          setDatabase(res.data);
-          reject();
+          if (res.data) {
+            setDatabase(res.data);
+            reject();
+          } else {
+            reject();
+
+            throw jsonData['api-error-messages']['unexpected-server-response'];
```

Comment:
What happens in the UI when this error is thrown? I don't think this is caught anywhere in the component.

Reply:
@vivekratnavel , we are catching the error in catch block.